tests/test_backend.py

```diff
@@ -591,9 +591,9 @@ def test_ttl(self):
         ttl = cache.ttl("foo")
 
         if isinstance(cache.client, herd.HerdClient):
-            self.assertAlmostEqual(ttl, 12)
+            self.assertAlmostEqual(ttl, 12, delta=0.001)
```

Comment:
This should be removed in favor of rounding the number of seconds returning in the function.

Reply:
After rounding the response from pttl and removing delta, I'm getting 
```
AssertionError: 9.999 != 10 within 7 places (0.0009999999999994458 difference)
```
Since the value of places defaults to 7. This seems to be a floating point precision problem, thus supplying delta seems necessary here.

Reply:
However, that leads to errors when comparing values like,
```AssertionError: 5.499 != 5.5 within 0.001 delta (0.001000000000000334 difference)```

Reply:
This is the logic with rounding the value returned from redis
```
t = client.pttl(key)
...
round(t / 1000, 3)
```

Reply:
I think delta should remain and a larger delta is needed, of course the code is right but we cannot control the delay between the check and redis response at the millisecond precision because also everybody does not have the biggest resources to get close to it.